fetch.bs

```diff
@@ -1661,6 +1662,10 @@ Unless stated otherwise, it is unset.
 
 <p class="note no-backref">This flag is for exclusive use by HTML's navigate algorithm. [[!HTML]]
 
+<p>A <a for=/>request</a> has an associated
+<dfn export for=request id=concept-request-user-activation-flag>user-activation flag</dfn>. Unless
+stated otherwise, it is unset.
```

Comment:
Let's change this into a boolean, similar to keepalive. I'd like to avoid adding flags and slowly convert existing flags to booleans. (I'm happy to help with this for the Fetch and HTML PRs if needed, but would prefer if someone else did it for Fetch Metadata.)

I think we should also add a note that this is for exclusive usage by HTML's navigate algorithm. At least for now?

Reply:
Updated in https://github.com/whatwg/fetch/pull/993/commits/c0050629590f01932fcaf4931699705c54029daa. I'll take care of the HTML PR and Fetch Metadata once you're happy with this patch.

Reply:
HTML is addressed in the updates to https://github.com/whatwg/html/pull/5203.

Fetch Metadata is renamed as well: https://w3c.github.io/webappsec-fetch-metadata/#abstract-opdef-append-the-fetch-metadata-headers-for-a-request.